Docker automation git action (#27)
badgerloop-software/chase-car-dashboard: README.md

4. Run `npm i` to install the latest dependencies to your computer.
5. In a terminal, run `npm start` or, if you want to run the engineering dashboard without the solar car dashboard, `npm run start-dev`. The console output may look messy, but that is due to three servers running at the same time. For a more organized console, please look below.

## Running the Dashboard With Docker

> (Not for developing)

Reviewer: We could leave this out. I know I've mentioned that this probably won't be used for development outside of making sure any changes we make work with Docker, but we don't need to explicitly say that it's not for development.
```suggestion
```

Reviewer 2: Got it! Just changed it!

> 0. Install Docker Desktop or docker. If you are not familiar with docker, it is recommended to install to install Docker Desktop.

Reviewer: We should add links to instructions for any required installations so that people know that they are installing the right thing and don't have to search for it.
Also, some miscellaneous changes.
```suggestion
0. Install [Docker Desktop](LINK TO DOWNLOAD/INSTALLATION INSTRUCTIONS) or [Docker](LINK TO DOWNLOAD/INSTALLATION INSTRUCTIONS).
   1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
```

Reviewer 2: updated

> 0. Install Docker Desktop or docker. If you are not familiar with docker, it is recommended to install to install Docker Desktop.

Reviewer: Food for future thought. This can be taken care of outside of the PR:
We should have two different sets of run instructions: one for Docker Desktop and one for Docker CLI. In addition, it would be a good idea to have visuals for the Docker Desktop instructions, in which case, we should create a Confluence page with more detailed instructions for both methods and just link that Confluence page in this section of the README.

> 1. If you are not familiar with docker, it is recommended to install Docker Desktop.

Reviewer: ```suggestion
0. Install [Docker Desktop](https://docs.docker.com/get-docker/) or [Docker](https://docs.docker.com/engine/install/). 
1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
```

Author: fixed!

Reviewer 2: updated

> 2. Once it is installed, start Docker Desktop. If docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.

Reviewer: ```suggestion
2. Once it is installed, start Docker Desktop. If Docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.
```
Sorry, missed one in my last review

Reviewer 2: updated!

1. Once it is installed, start Docker Desktop. If docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.

> 2. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called windows-pull-run.bat, and for linux and mac it is called linux-pull-run.sh.

Reviewer: ```suggestion
2. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called `windows-pull-run.bat`, and for Linux and Mac it is called `linux-pull-run.sh`.
```

Reviewer 2: Updated!

> 3. Download the script and run it (double click on it). Or, you can copy the contents of the script and paste it into a terminal or command prompt. 

Reviewer: For people who aren't familiar with GitHub, we should have brief instructions on how to download the script from GitHub.
Also, minor change.
```suggestion
3. Download the script and run it (double click on it), or you can copy the contents of the script and paste it into a terminal or command prompt. 
```

Reviewer 2: updated

> 5. To download the script, open the script in Github and then click "Raw". The file will open in a tab. Then do control+s or command+s depending on your OS and the script will be downloaded.

Reviewer: ```suggestion
4. Download the script (see bullet point) and run it (double click on it), or you can copy the contents of the script and paste it into a terminal or command prompt. 
   - To download the script, open the script in Github and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and the script will be downloaded.
```

Author: fixed!

Reviewer 2: updated

>    - To download the script, open the script in Github and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and the script will be downloaded.

Reviewer: ```suggestion
   - To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and the script will be downloaded.
```

Reviewer 2: updated

5. In a terminal, run `npm start` or, if you want to run the engineering dashboard without the solar car dashboard, `npm run start-dev`. The console output may look messy, but that is due to three servers running at the same time. For a more organized console, please look below.

## Running the Dashboard With Docker

0. Install [Docker Desktop](https://docs.docker.com/get-docker/) or [Docker](https://docs.docker.com/engine/install/). 
1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
2. Once it is installed, start Docker Desktop. If docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.
3. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called `windows-pull-run.bat`, and for Linux and Mac it is called `linux-pull-run.sh`.
4. To download the script, open the script in Github and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, save it to your Downloads folder, and the script will be downloaded. To run, look at the bullet points below:

>    - (Windows) Run these commands in your command prompt `cd Downloads`, `move windows-pull-run.bat.txt windows-pull-run.bat`, and `windows-pull-run.bat`. 

Reviewer: To keep out of the terminal, I would just say to remove ".txt" when saving the file. Then, you can just double-click it in the file explorer.

Author: updated
